src/avrdude.conf.in

```diff
@@ -1458,6 +1458,23 @@ programmer
     usbpid                 = 0x0c9f;
 ;
 
+#------------------------------------------------------------
+# gemmaBoot
+#------------------------------------------------------------
+
+# Serving the Gemma bootloader
+# https://github.com/adafruit/Adafruit-Trinket-Gemma-Bootloader
+
+programmer
+    id                     = "gemmaBoot";
+    desc                   = "Gemma bootloader disguised as USBtiny";
+    type                   = "usbtiny";
+    prog_modes             = PM_SPM;
+    connection_type        = usb;
+    usbvid                 = 0x2341;
+    usbpid                 = 0x0105;
```

Comment:
For Arduino Gemma Bootloader
usbpid = 0x2341
usbpid = 0x0c9f

For Adafruit Trinket Gemma Bootloader
usbpid = 0x1781
usbpid = 0x0c9f